docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+db:
+  image: postgres:9.4
+  ports:
+   - "5432"
+web:
+  build: .
```

Comment:
Yeah, I like what you've been doing there. I'd like to do that as an additional change in a new pull request once we get the image building and deployed.